src/GraphWebhooks/Graph/ChangeNotificationEncryptedContentExtensions.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using Microsoft.Graph.Models;
+using Microsoft.Kiota.Abstractions.Serialization;
+
+namespace GraphWebhooks;
+
+// This extension is to workaround this bug in the SDK:
+// https://github.com/microsoftgraph/msgraph-sdk-dotnet/issues/2237
+// Should no longer be needed after that bug is fixed.
+// This is a direct copy of
+// https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blob/dev/src/Microsoft.Graph.Core/Extensions/IDecryptableContentExtensions.cs
+public static class ChangeNotificationEncryptedContentExtensions
+{
+    public static async Task<T?> DecryptAsync<T>(
```

Comment:
should accept and pass down a cancellationToken

Reply:
You're right, but I'd point out this is a copy of the Core SDK's code ;)

Reply:
why are we copying this if this is already in core?

Reply:
Because SDK bugs: https://github.com/microsoftgraph/msgraph-sdk-dotnet/issues/2237

I have a lot of code in here that can go once that bug is fixed. The whole ChangeNotificationEncryptedContentExtensions.cs can go, along with EncryptableSubscription.cs.

Reply:
thanks for pointing this out, I had forgotten about it. Resolving since you've passed a cancellation token.